apps/files/src/store/actions.js

```diff
@@ -248,6 +278,27 @@ export default {
       context.commit('UPDATE_FOLDER_LOADING', false)
     })
   },
+  loadFolderSharedFromMe (context, { client, $gettext }) {
+    context.commit('UPDATE_FOLDER_LOADING', true)
+
+    client.requests.ocs({
```

Comment:
are we missing an api method in the sdk here?

Reply:
I found mentions of shared_with_me in the shares there but no way how to load shared with others. (tried to use the getShares function, it's possible I overlooked something or did something wrong though)